styles.css

```diff
@@ -0,0 +1,389 @@
+  margin-top: 30px;
+}
+
+.head-work {
+  display: flex;
+  flex-direction: column;
+  min-height: 80vh;
+  margin-top: 50px;
+}
+
+.head-work img {
+  width: 80%;
+  align-self: center;
+}
+
+.head-work h2 {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.recent-works p {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.techstack-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+}
+
+.techstack-container li {
+  font-family: 'Inter', sans-serif;
+  border: 1px solid #8993a4;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  list-style-type: none;
+}
+
+.head-work .project-button {
+  background-color: #ff6b00;
+  height: 48px;
+  color: #fff;
+  align-self: flex-start;
+  margin-left: 36px;
+  margin-top: 20px;
+  padding: 12px;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  font-family: 'Inter', sans-serif;
+}
+
+.work .project-button {
+  background-color: #ff6b00;
+  height: 48px;
+  color: #fff;
+  align-self: left;
+  margin-top: 22px;
+  margin-left: 1.5px;
+  padding: 12px;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  font-family: 'Inter', sans-serif;
+  width: 99%;
+  border: none;
+}
+
+.head-work .project-button:hover {
+  background: rgba(0, 0, 0, 0.3);
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.works {
+  display: grid;
+  background-color: #fff;
+  row-gap: 10px;
+}
+
+.work {
+  display: flex;
+  flex-direction: column;
+  max-height: 80vh;
+  background-image: url(assets/Img\ Placeholder1.png);
+  background-repeat: no-repeat;
+  background-size: 90%;
+  padding-left: 16px;
+  padding-right: 16px;
+  background-position: top;
+}
+
+.work h2 {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  width: 80%;
+  color: #fff;
+}
+
+.work p {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.work .techstack-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 36px;
+  max-width: 80%;
+  margin-top: 10px;
+}
+
+.work li {
+  color: #fff;
+}
+
+.work .project-button:hover {
+  background: rgba(0, 0, 0, 0.3);
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.about {
+  background-color: #1c1a19;
+  min-height: 100vh;
+  display: flex;
```

Comment:
Please kindly type **display: flex** on the top to make it semantically correct 👍